ui/src/index.scss

```diff
@@ -27,3 +27,19 @@ $sm: 700px;
   --off-white: #f6f7fe;
   --input-bg: #bcd5eb36;
 }
+
+.dropdown__control {
+  background-color: transparent !important;
+  border: none !important;
+}
+
+.dropdown__single-value {
+  color: var(--secondary-color) !important;
+}
+.dropdown__indicator-separator {
+  background: transparent !important;
+}
+
+.dropdown__menu {
+  color: var(--primary-text) !important;
+}
```

Comment:
Is it necessary to use `!important`?

Reply:
yes, because we are overriding the default styles of the dropdown.

Reply:
Did you try this: https://react-select.com/styles#using-classnames?

I believe this should help us avoid `!important`.

Reply:
> Did you try this: https://react-select.com/styles#using-classnames?

This is what I am using in this PR.

Reply:
Oh my bad. I missed that part.

Clickable part is not the complete button.

![image](https://user-images.githubusercontent.com/15676805/180228555-43150308-5f3e-419b-b7fd-d40e7a11a3e5.png)